util/test/paratest.client

```diff
@@ -53,7 +53,7 @@ sub main {
 
     if ($#ARGV < 6) {
         fatal("insufficient arguments: '@ARGV'\n" .
-	      "Usage:  paratest.client id chapeltestdir testdir chplenv futures-mode valgrind memleaks [compopts [execopts]]");
+	      "Usage:  paratest.client id chapeltestdir testdir chplenv futures-mode valgrind memleaks memleakslog [compopts [execopts]]");
```

Comment:
I might follow what we do for the valgrind/futures args and have it be something like memleaks=0 is no mem leaks, 1 for --memLeaks and 2 for --memLeaksLog (only because I always screw something up when adding a new arg to paratest, so if we can use an existing arg I think that increases the chance of us getting it right)

Reply:
Yeah, that makes sense. Will do that.